appium/webdriver/extensions/android/activities.py

```diff
@@ -12,21 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import TypeVar
+import warnings
+from typing import TYPE_CHECKING, cast
 
-from selenium.common.exceptions import TimeoutException
+from selenium.common.exceptions import TimeoutException, UnknownMethodException
 from selenium.webdriver.support.ui import WebDriverWait
 
 from appium.protocols.webdriver.can_execute_commands import CanExecuteCommands
+from appium.protocols.webdriver.can_execute_scripts import CanExecuteScripts
+from appium.protocols.webdriver.can_remember_extension_presence import CanRememberExtensionPresence
 from appium.webdriver.mobilecommand import MobileCommand as Command
 
-T = TypeVar('T', bound=CanExecuteCommands)
+if TYPE_CHECKING:
+    from appium.webdriver.webdriver import WebDriver
 
 
-class Activities(CanExecuteCommands):
-    def start_activity(self: T, app_package: str, app_activity: str, **opts: str) -> T:
+class Activities(CanExecuteCommands, CanExecuteScripts, CanRememberExtensionPresence):
+    def start_activity(self, app_package: str, app_activity: str, **opts: str) -> 'WebDriver':
         """Opens an arbitrary activity during a test. If the activity belongs to
         another application, that application is started and the activity is opened.
+        deprecated:: 2.0.0
```

Comment:
Does this 2.0.0 mean this client library?
Current one is 2.9.0, so 3.0.0?

Reply:
I was not quite sure what to put there. I assume it should be the current version, so will change it to 2.9.0

Reply:
Ah, gotcha